int32 arg1
int32 arg2
---
int32 resval
